src/styles/blocs/header.scss

```
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  height: $header-height;
+  background-color: #fff;
```

make variables for all colors
```suggestion
  background-color: $c-white;
```